modules/dhcp/dhcp_api.rb

```diff
@@ -53,8 +56,8 @@ class Proxy::DhcpApi < ::Sinatra::Base
 
   # returns an array of records for an ip address
   get "/:network/ip/:ip_address" do
+    server.validate_supported_address(params[:ip])
```

Comment:
Here you need to test both - for this route, Sinatra stores two parameters named `:network` and `:ip_address`. Check both. There is no parameter named `:ip`.